Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -33,6 +33,20 @@ export function findAliasIdsForDocumentSql(documentId: string): string {
   return format(`SELECT alias_id FROM meadowlark.aliases WHERE document_id = %L FOR SHARE NOWAIT`, documentId);
 }
 
+/**
+ * Returns the SQL query to find the existence of an alias id. Alias ids include the document id
+ * itself along with any superclass variations that the document satisifies. For example, a School is a subclass
+ * of EducationOrganization, so each School document has an additional alias id as an EducationOrganization.
+ *
+ * This function does NOT read-for-write lock.
```

Comment:
I agree with that. There is a tiny possibility of two competing clients racing to post a record first - unless we lock the entire table, we can't prevent that. So be it.